.gitignore

```diff
@@ -6,6 +6,7 @@ tmp/*
 *.dump
 .DS_Store
 public/assets
+*.dump
```

Comment:
this duplicates line 6

Reply:
Yeah, I'm not sure why it's doing that. Dump files are included in my git ignore...

Reply:
That's because I added a commit that added the line to the `.gitignore`, and Michiel added another one to `master`, which you then merged into this branch :)